src/components/og-spinner/readme.md

```diff
@@ -0,0 +1,55 @@
+# og-spinner
+
+
+
+<!-- Auto Generated Below -->
+
+
+## Properties
+
+| Property   | Attribute  | Description                                        | Type      | Default     |
+| ---------- | ---------- | -------------------------------------------------- | --------- | ----------- |
+| `disabled` | `disabled` | Determines, whether the control is disabled or not | `boolean` | `undefined` |
+| `label`    | `label`    | The label of the spinner                            | `string`  | `undefined` |
```

Comment:
I think the readme needs to be re-generated, since we can find non-existing properties here (`label`) and others are missing (`size`). Same goes for the events.